plugin/zoom.vim

```diff
@@ -6,5 +6,9 @@ let g:loaded_zoom = 1
 nnoremap <silent> <Plug>(zoom-toggle) :call zoom#toggle()<CR>
 
 if !hasmapto('<Plug>(zoom-toggle)')
-  nmap <C-W>m <Plug>(zoom-toggle)
+  if $TMUX == ''
+    nmap <C-W>m <Plug>(zoom-toggle)
+  else
+    nmap <C-W>z <Plug>(zoom-toggle)
+  endif
```

Comment:
```suggestion
  nmap <C-W>m <Plug>(zoom-toggle)
  if empty($TMUX)
    nmap <C-W>z <Plug>(zoom-toggle)
  endif
```

Perhaps we can retain the regular mapping, and optionally introduce the `<C-W>z` mapping additionally when tmux is detected.